core/commands/pin/remotepin.go

```diff
@@ -366,11 +411,26 @@ collected if needed.
 var addRemotePinServiceCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
 		Tagline:          "Add remote pinning service.",
-		ShortDescription: "Add a credentials for access to a remote pinning service.",
+		ShortDescription: "Add credentials for access to a remote pinning service.",
+		LongDescription: `
+Add credentials for access to a remote pinning service and store them in the
+config under Pinning.RemoteServices map.
+
+TIP:
+
+  To add services and test them by fetching pin count stats:
+
+  $ ipfs pin remote service add goodsrv https://pin-api.example.com secret-key
+  $ ipfs pin remote service add badsrv  https://bad-api.example.com invalid-key
+  $ ipfs pin remote service ls --stat
+  goodsrv   https://pin-api.example.com 0/0/0/0
+  badsrv    https://bad-api.example.com invalid
+
+`,
 	},
 	Arguments: []cmds.Argument{
 		cmds.StringArg(pinServiceNameOptionName, true, false, "Service name."),
-		cmds.StringArg(pinServiceURLOptionName, true, false, "Service URL."),
+		cmds.StringArg(pinServiceEndpointOptionName, true, false, "Service endpoint."),
 		cmds.StringArg(pinServiceKeyOptionName, true, false, "Service key."),
```

Comment:
These are arguments not options, so we should name the variables `ArgumentName` instead of `OptionName`

Reply:
Done: 4e2d17fd7fdca179f167812fd344299e27c3491d